tests/Core/TimerUnitTests.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
Can you remove this file? I had removed in a previous check-in.

Reply:
I shall do so